src_c/image.c

```diff
@@ -1252,6 +1252,15 @@ image_frombuffer(PyObject *self, PyObject *arg)
 
         */
     }
+    else if (!strcmp(format, "BGR")) {
+        if (len != (Py_ssize_t)w * h * 3)
+            return RAISE(
+                PyExc_ValueError,
+                "Buffer length does not equal format and resolution size");
+        surf = SDL_CreateRGBSurfaceFrom(data, w, h, 24, w * 3,
+                                        0xFF << 16, 0xFF << 8,
```

Comment:
I guess this might need a `SDL_BYTEORDER == SDL_LIL_ENDIAN` block?

Reply:
I was going to add one but noticed it had been commented out for the RGB block above here:

https://github.com/pygame/pygame/blob/16a8d3ea36ff09fc1c1d91c6492219a4bef944db/src_c/image.c#L1240-L1254

And thought that maybe this had been removed on purpose and I should follow suit.

Reply:
This can be tested later once the big endian test bots are setup.